src/supported-value.js

```diff
@@ -1,11 +1,26 @@
 import isInBrowser from 'is-in-browser'
 import prefix from './prefix'
+import supportedProperty from './supported-property'
 
 const cache = {}
+const transitionProperties = [
```

Comment:
I am wondering if transition is unique in this, maybe we need a more general solution for all similar cases.